components/product-listing/product.css

```diff
@@ -0,0 +1,281 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    background-color: var(--lt-cream);
+    font-family: 'Arima Madurai', cursive;
+
+}
+
+header {
+    z-index: 5;
+    padding: 1rem;
+    top: 0;
+    position: sticky;
+    width: 100%;
+    background-color: var(--dk-cream);
+}
+
+.flex-sa {
+    display: flex;
+    justify-content: space-around;
+    align-items: center
+}
+
+.flex-gap {
+    gap: 0.1rem;
+}
+
+.block {
+    display: inline-block;
+}
+
+.vilvah-green {
+    color: var(--md-green);
+}
+
+.arima_madurai {
+    font-family: 'Arima Madurai', cursive;
+}
+
+.arima_madurai_100 {
+    font-family: 'Arima Madurai', cursive;
+    font-weight: 100;
+}
+
+.input-items {
+    padding: 0.6rem;
+    padding-right: 4rem;
+    border-radius: 1rem;
+    border: none;
+    /* color: var(--dk-grey); */
+    box-shadow: 0 1px 8px #00000040;
+    width: 30%;
+    font-weight: 500;
+    padding-left: 1.5rem;
+    border-style: none;
+    outline: none;
```

Comment:
Outline none creates an accessibility issue. 

Instead do

```css
outline: 2px solid transparent;
outline-offset: 2px;
```

Reply:
ok thahir i'll change it